docs/modules/ROOT/pages/documentation/payara-server/jbatch/asadmin.adoc

```diff
@@ -116,11 +116,42 @@ the classes mentioned:
 </dependency>
 ----
 
-_Since Payara Server 5.183_
+[[clean-jbatch-repository]]
+== `clean-jbatch-repository`
+
+_Since Payara Server 5.25.0_
+
+*Usage*::
+`asadmin> clean-jbatch-repository --status COMPLETED|ALL --days <int.value> <JobName>
+
+*Aim*::
+The clean-jbatch-repository subcommand deletes records of job executions
+
+[[command-options-clean]]
+=== Command Options
+
+[cols=",,,,",options="header",]
+|=======================================================================
+|Option | Type |Description |Default |Mandatory
+|`--status` | ALL/COMPLETED| The status of batch jobs executions to which delete the records of.
+If ALL then all job executions will be deleted, regardless of status.
+If COMPLETED only job executions that successfully completed will be deleted. | COMPLETED |No
+|`--days` | Integer| The number of days previous that the end time of the job execution must be to deleted. | 0 | No
+|=======================================================================
+
+[[example-clean]]
+===Example
+
```

Comment:
It might be a bit obvious, but add a short description to this example like "The following command will delete all Batch jobs that were executed before 7 days ago"